src/index.ts

```diff
@@ -147,11 +187,70 @@ export type PlatformApi<Platform extends SupportedPlatform, RunTarget extends Su
      */
     setClipboard: (text: string) => Promise<void>;
 
+    /**
+     * Install the certificate authority with the given path as a trusted CA on the device. This allows you to intercept
+     * and modify traffic from apps on the device.
+     *
+     * On Android, this installs the CA as a system CA. As this is normally not possible on Android 10 and above, it
+     * overlays the `/system/etc/security/cacerts` directory with a tmpfs and installs the CA there. This means that the
+     * changes are not persistent across reboots.
+     *
+     * On iOS, the CA is installed permanently as a root certificate in the Certificate Trust Store. It persists across
+     * reboots.\
+     * **Currently, you need to manually trust any CA at least once on the device, CAs can be added but not
+     * automatically marked as trusted (see:
+     * https://github.com/tweaselORG/appstraction/issues/44#issuecomment-1466151197).**
+     *
+     * Requires the `root` capability on Android, and the `ssh` capability on iOS.
+     *
+     * @param path The path to the certificate authority to install. The certificate must be in PEM format.
+     */
+    installCertificateAuthority: (path: string) => Promise<void>;
+    /**
+     * Remove the certificate authority with the given path from the trusted CAs on the device.
+     *
+     * On Android, this works for system CAs, including those pre-installed with the OS. As this is normally not
+     * possible on Android 10 and above, it overlays the `/system/etc/security/cacerts` directory with a tmpfs and
+     * removes the CA there. This means that the changes are not persistent across reboots.
+     *
+     * On iOS, this only works for CAs in the Certificate Trust Store. It does not work for pre-installed OS CAs. The
+     * changes are persistent across reboots.
+     *
+     * Requires the `root` capability on Android, and the `ssh` capability on iOS.
+     *
+     * @param path The path to the certificate authority to remove. The certificate must be in PEM format.
+     */
+    removeCertificateAuthority: (path: string) => Promise<void>;
+    /**
+     * Set or disable the proxy on the device. If you have enabled the `wireguard` capability, this will start or stop a
+     * WireGuard tunnel. Otherwise, it will set the global proxy on the device.
+     *
+     * WireGuard is currently only supported on Android. Enabling a WireGuard tunnel requires the `root` capability.
+     *
+     * @remarks
+     * The WireGuard integration will create a new tunnel in the app called `appstraction` and delete it when the proxy
+     * is stopped. If you have an existing tunnel with the same name, it will be overridden.
+     * @param proxy The proxy to set, or `null` to disable the proxy. If you have enabled the `wireguard` capability,
+     *   this is a string of the full WireGuard configuration to use.
+     */
```

Comment:
I was just think as well, that we should mention that we set the proxy in iOS only for a WiFi connection and not cellular data. I know it is kinda ridiculous, because the method requires the `ssh` capability and it is unlikely this works over cellular, but I think this should still be mentioned, because we don't know the purposes someone might want to use this for. 

Also, I looked into WireGuard on iOS for a moment and they seem to support VPNs on cellular as well, so if we want to enable WireGuard on iOS we could support cellular, too.